recipes/fmilib/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.0.2" %}
+
+package:
+  name: fmilib
+  version: {{ version }}
+
+source:
+  fn: FMILibrary-{{ version }}-src.zip
+  url: http://www.jmodelica.org/downloads/FMIL/FMILibrary-{{ version }}-src.zip
+  sha256: 85ff3358833818103d05e6d45f3fffcd2bbced9ef5f2262acac4d2de6523356e
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - python  # [win]
+
+test:
```

Comment:
Please add the following. This should fix the build failure on AppVeyor.

```yaml

test:
  requires:
    - python {{ environ['PY_VER'] + '*' }}  # [win]
```

xref: https://github.com/conda/conda-build/issues/835